src/ContainerHealthLogs.jsx

```diff
@@ -35,12 +35,20 @@ const format_nanoseconds = (ns) => {
     return cockpit.format(cockpit.ngettext("$0 second", "$0 seconds", seconds), seconds);
 };
 
+const HealthcheckOnFailureActionText = {
+    none: _("No action"),
+    restart: _("Restart"),
+    stop: _("Stop"),
+    kill: _("Force stop"),
+};
```

Comment:
Ah right, the API wants integers for creation, and returns strings when we request it...

Reply:
when do you think API will change the returned string?  :)